### DISCUSSION TITLE
Unable to connect to server

### DISCUSSION THREAD

**Discussion by pyeager56** [bluecherrydvr/unity#52] [General]
using IP address and port 7001

Tried admin account and a view-only account.

Same result whether at program startup, or if I skip that and attempt to add server once the program has started.

![unity](https://user-images.githubusercontent.com/106773418/214720041-f1f4600e-cce6-4eb4-a0ad-1424891ad5f6.PNG)

**Comment by curtishall** [comment]
This is fixed in #43 can you download beta2 and try that?

**Comment by pyeager56** [comment]
I’m guessing what you suggest I download is not what I downloaded last night.

I’m not a github expert.  Where would I find this file?

Paul Yeager
IT Manager
Cataloochee Ski Area
#thinkwinter
828 926 0285 ext 7321
800 768 0285 ext 7321
828 926 0354 fax
828 400 9442 cell

**Comment by pyeager56** [comment]
Scrolling down to download on this page:

https://github.com/bluecherrydvr/unity

When I click Windows, I get a 404.


Paul Yeager
IT Manager
Cataloochee Ski Area
#thinkwinter
828 926 0285 ext 7321
800 768 0285 ext 7321
828 926 0354 fax
828 400 9442 cell

**Reply by curtishall** [reply]
https://github.com/bluecherrydvr/unity/releases

or - https://github.com/bluecherrydvr/unity/releases/tag/v3.0.0-beta2

**Comment by pyeager56** [comment]
Thank you.  I can add the server now.

I added an issue for camera appearing blank when added to view, until another camera is added.

The device list at the left of the window is almost ordered by device ID.  It would be helpful to have an option for the list to be alphabetically sorted by device name.  The makes the task of finding a specific device easier when you have over 100 devices.

Thanks,

Paul Yeager
IT Manager
Cataloochee Ski Area
#thinkwinter
828 926 0285 ext 7321
800 768 0285 ext 7321
828 926 0354 fax
828 400 9442 cell

**Reply by curtishall** [reply]
> It would be helpful to have an option for the list to be alphabetically sorted by device name. The makes the task of finding a specific device easier when you have over 100 devices.

Added - https://github.com/bluecherrydvr/unity/issues/57
